Header header
uint16 object_type  #0 for field, 1 for ball, 2 for goalpost, 3 for other
float64[] image_u
float64[] image_v
float64 object_range
float64 object_bearing
float32 head_pitch
float64 object_width
#float64 object_height
